### PR TITLE
Windows tests

### PR DESCRIPTION
**Pull request by MikeLaptev**
Hi Timur,

This pull request contains:
1. Cosmetic changes in docstrings to have the same format across all common functions
2. Minor fix in common functionality and some minor improvements
3. Base point for tests on windows compatibility, just to make sure that it is in progress
- \- I've marked such scenarious as not implemented, because all of them is under development, but commited functinality works fine. And I don't see any reason to keep it on local branch.
  *\* - I've additional 2 lines in test.sh just to separate execution of the 'Basic Nova Verification' tests from '
  'Windows Compatibility' tests

Thanks,
Mike

### CODE REVIEW

File: mos_tests/nova/windows_compatibility_tests.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+                    print "Node in the {} state".format(self.node_to_boot.status)
+                    if self.node_to_boot.status != 'BUILD':
+                        is_created = True
+                        break
+            time.sleep(5)
+        # check that boot returns expected results
+        self.assertEqual(self.node_to_boot.status, 'ACTIVE',
+                         "The node not in active state!")
+
+        # TODO: test is here
```
Comment (TimurNurlygayanov) [inline mos_tests/nova/windows_compatibility_tests.py:226]: ))) ok, let's wait for the test

File: mos_tests/functions/common.py [hunk 2]
```diff
@@ -33,14 +32,15 @@ def check_stack_status(stack_name, heat, status, timeout=60):
...
+                    False otherwise
     """
     if check_stack(stack_name, heat):
         start_time = time()
         stack_status = [s.stack_status for s in heat.stacks.list()
-                    if s.stack_name == stack_name][0]
-        while stack_status.find('IN_PROGRESS') != -1 and time() < \
-                        start_time + 60 * timeout:
+                        if s.stack_name == stack_name][0]
+        while stack_status.find('IN_PROGRESS') != -1 \
```
Comment (gdyuldin) [inline mos_tests/functions/common.py]: IMHO `'IN_PROGRESS' not in stack_status` looks  better.
Also you can use https://pypi.python.org/pypi/waiting for it.

Comment (MikeLaptev) [inline mos_tests/functions/common.py]: I got it.
What do you think about whole string for the status; 'CREATE_IN_PROGRESS'?
I tried to figure out list of available statuses, but currently without success...

Comment (MikeLaptev) [inline mos_tests/functions/common.py]: Please ignore my previous comment :)

**Comment by MikeLaptev** [comment]
I've closed this pull request

**Pull request closed without merging**